demo/Example+SLRealizer+Usage.ipynb

```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import absolute_import\n",
+    "from __future__ import division\n",
+    "from __future__ import print_function\n",
+    "\n",
+    "import os, sys\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "realizer_path = os.path.join(os.environ['SLREALIZERDIR'], 'slrealizer')\n",
+    "sys.path.insert(0, realizer_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to interface with SLRealizer\n",
+    "\n",
+    "Author: Ji Won Park (jiwoncpark)\n",
+    "\n",
+    "This notebook demonstrates how to create an SLRealizer instance and use its methods to emulate the LSST DRP Source and Object catalogs (\"source and object tables\"). We will use the OM10Realizer subclass to realize lensed quasar systems from OM10 (Oguri and Marshall 2010) and the SDSS subclass to realize galaxies from SDSS DR14 (Abolfathi et al. 2018).\n",
+    "\n",
+    "To use SLRealizer, we need two types of catalogs: one defining the static properties of objects such as size and flux (\"object catalog\") and another defining the observation conditions such as atmospheric seeing and the cadence schedule (\"observation catalog\"). The subclasses of SLRealizer are named after the object catalog; OM10Realizer realizes objects in the OM10 mock quasar catalog and SDSSRealizer realizes objects in SDSS data release products.\n",
+    "\n",
+    "We will use the same observation catalog in our realization of lenses and non-lenses. Called `minion_1016`, this is the current official Reference Simulated Survey for the LSST. Let's begin by defining the path to our observation catalog."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is the directory where all our data files will be stored.\n",
+    "data_dir = os.path.join(os.environ['SLREALIZERDIR'], 'data')\n",
+    "# Observation catalog filename\n",
+    "observation_fname = os.path.join(data_dir, 'twinkles_observation_history.csv')\n",
+    "# Observation catalog dataframe\n",
+    "observation_df = pd.read_csv(observation_fname).query(\"(filter != 'y')\").reset_index(drop=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note I have queried out entries with the y-band; this is simply because the SDSS does not have a y-band!"
+   ]
+  },
+  {
```

Comment:
The above should go in a "Requirements" section, and I think that should probably also describe how to get he `OM10` and `LensPop` packages as well (since that is somewhat non-trivial). I don't think it's as easy as `pip install -r requirements.txt` is it?

Reply:
I guess the installation instructions should be in the README page. The page needs other updates as well.

The requirements will be easier to investigate once we have Travis up and running. My Travis had trouble compiling with gcc when downloading GalSim, so OM10 and LensPop may not be the only problems. (I actually assumed they would be okay?)

Reply:
OK, fair enough - can you issue this work, please? Thanks!